Homework4/ArrFunc_TS6.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <iostream>
+
+void rInitArray(int *const array_begin, int const *const array_end);
+void fInitArray(int *const array_begin, int const *const array_end);
+void uInitArray(int *const array_begin, int const *const array_end);
+void ShowArray(int *const array_begin, int const *const array_end);
+
+const size_t k_int_array_size{10'000};
+const int k_min_random_value{10};
+const int k_max_random_value{99};
+const char k_file_name[]{"Taras_Samchuk_HW4_Task_1.txt"};
```

Comment:
такие константы лучше выносить в cpp файле где они используется, это лучше дасть читабельнсоть кода
```suggestion
static const size_t k_int_array_size{10'000};
static const int k_min_random_value{10};
static const int k_max_random_value{99};
static const char k_file_name[]{"Taras_Samchuk_HW4_Task_1.txt"};
```